src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level


see common mistakes https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery

Reply:
do you mean that 
private final Random random;